k8s/elasticsearch/README.md

```diff
@@ -127,13 +127,23 @@ Specify the number of replicas for the Elasticsearch cluster:
 ```shell
 export REPLICAS=2
 ```
+Enable Stackdriver Metrics Exporter:
+
+ > **NOTE:** Your GCP project should have Stackdriver enabled. For non-GCP clusters, export of metrics to Stackdriver is not supported yet.
+By default the integration is disabled. To enable, change the
+METRICS_EXPORTER_ENABLED variable value to `true`.
+
+ ```shell
+export METRICS_EXPORTER_ENABLED=true
```

Comment:
Please change this to false - Other solutions default to `false` and it would be good to keep common part the same.